CHANGELOG.md

```diff
@@ -0,0 +1,44 @@
+## v1.0.3
```

Comment:
I mostly just pulled the existing release notes from an open draft and the previous releases to create this file. Going forward, we'll have to manually maintain this. We can probably mimic what Matt's old team did and update this as we go (the publication workflow only triggers when the `version.txt` file is updated, not this change log)